graylog2-server/src/test/java/org/graylog2/contentpacks/facades/InputFacadeTest.java

```diff
@@ -338,12 +369,89 @@ public void delete() throws NotFoundException {
+        when(lookupTableDto.id()).thenReturn("dead-beef");
+        final Input input = inputService.find("5ae2eb0a3d27464477f0fd8b");
+        final InputWithExtractors inputWithExtractors = InputWithExtractors.create(input, inputService.getExtractors(input));
+        final LookupTableExtractor extractor = (LookupTableExtractor) inputWithExtractors.extractors().iterator().next();
+        final ExtractorEntity extractorEntity = ExtractorEntity.create(
+                ValueReference.of(extractor.getTitle()),
+                ValueReference.of(extractor.getType()),
+                ValueReference.of(extractor.getCursorStrategy()),
+                ValueReference.of(extractor.getTargetField()),
+                ValueReference.of(extractor.getSourceField()),
+                ReferenceMapUtils.toReferenceMap(extractor.getExtractorConfig()),
+                Collections.emptyList(),
+                ValueReference.of(extractor.getConditionType()),
+                ValueReference.of(extractor.getConditionValue()),
+                ValueReference.of(extractor.getOrder())
+        );
+        List<ExtractorEntity> extractors = new ArrayList();
+        extractors.add(extractorEntity);
+        InputEntity inputEntity = InputEntity.create(
+                ValueReference.of(input.getTitle()),
+                ReferenceMapUtils.toReferenceMap(input.getConfiguration()),
+                Collections.emptyMap(),
+                ValueReference.of(input.getType()),
+                ValueReference.of(input.isGlobal()),
+                extractors);
+        final Entity entity = EntityV1.builder()
+                .id(ModelId.of(input.getId()))
+                .type(ModelTypes.INPUT_V1)
+                .data(objectMapper.convertValue(inputEntity, JsonNode.class))
+                .build();
+        final LookupTableEntity lookupTableEntity = LookupTableEntity.create(
+                ValueReference.of("whois"),
+                ValueReference.of("title"),
+                ValueReference.of("description"),
+                ValueReference.of("cache_name"),
+                ValueReference.of("dataadapter_name"),
+                ValueReference.of("default_single_value"),
+                ValueReference.of("BOOLEAN"),
+                ValueReference.of("default_multi_value"),
+                ValueReference.of("BOOLEAN")
+        );
+        final Entity expectedEntity = EntityV1.builder()
+                .id(ModelId.of("dead-beef"))
+                .data(objectMapper.convertValue(lookupTableEntity, JsonNode.class))
+                .type(ModelTypes.LOOKUP_TABLE_V1)
+                .build();
+        final EntityDescriptor entityDescriptor = expectedEntity.toEntityDescriptor();
+        final Map<EntityDescriptor, Entity> entityDescriptorEntityMap = new HashMap<>(1);
+        entityDescriptorEntityMap.put(entityDescriptor, expectedEntity);
+        Graph graph = facade.resolveForInstallation(entity, Collections.emptyMap(), entityDescriptorEntityMap);
```

Comment:
Same as above, please use the generic type here instead of the raw type:

```suggestion
        Graph<Entity> graph = facade.resolveForInstallation(entity, Collections.emptyMap(), entityDescriptorEntityMap);
```